GitHubComments.ps1

```diff
@@ -0,0 +1,576 @@
+
+    .PARAMETER MediaType
+        The format in which the API will return the body of the comment.
+
+        raw - Return the raw markdown body. Response will include body. This is the default if you do not pass any specific media type.
+        text - Return a text only representation of the markdown body. Response will include body_text.
+        html - Return HTML rendered from the body's markdown. Response will include body_html.
+        full - Return raw, text and HTML representations. Response will include body, body_text, and body_html.
+
+    .PARAMETER MediaTypeVersion
+        The version of the response to return. The default version of the API may change in the future.
+        If you're building an application and care about the stability of the API, be sure to request 
+        a specific version. More info at https://developer.github.com/v3/media/.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Get-GitHubComment-OwnerName Powershell -RepositoryName PowerShellForGitHub
+
+        Get the comments for the PowerShell\PowerShellForGitHub project.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        DefaultParametersetName='RepositoryElements')]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification="Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
+    param(
+        [Parameter(Mandatory, ParameterSetName='RepositoryElements')]
+        [Parameter(Mandatory, ParameterSetName='IssueElements')]
+        [Parameter(Mandatory, ParameterSetName='CommentElements')]
+        [string] $OwnerName,
+
+        [Parameter(Mandatory, ParameterSetName='RepositoryElements')]
+        [Parameter(Mandatory, ParameterSetName='IssueElements')]
+        [Parameter(Mandatory, ParameterSetName='CommentElements')]
+        [string] $RepositoryName,
+
+        [Parameter(Mandatory, ParameterSetName='RepositoryUri')]
+        [Parameter(Mandatory, ParameterSetName='IssueUri')]
+        [Parameter(Mandatory, ParameterSetName='CommentUri')]
+        [string] $Uri,
+
+        [Parameter(Mandatory, ParameterSetName='CommentUri')]
+        [Parameter(Mandatory, ParameterSetName='CommentElements')]
+        [string] $CommentID,
+
+        [Parameter(Mandatory, ParameterSetName='IssueUri')]
+        [Parameter(Mandatory, ParameterSetName='IssueElements')]
+        [int] $IssueNumber,
```

Comment:
The `GitHubIssues.ps1` functions just use `Issue` as the parameter name.  I'd stay consistent.